config.example.toml

```diff
@@ -603,6 +603,13 @@ change-id = 116881
 # desired in distributions, for example.
 #rpath = true
 
+# Indicates whether `rustc` should be stripped of symbols using `-Cstrip=symbols`.
+#strip = false
```

Comment:
When is this necessary, if debuginfo is not enabled in the first place? Do we really need another option here?

Reply:
`-Cstrip=symbols ` will strip all symbols, not only the debuginfo. The artifacts will be smaller and have fewer symbols if we use `strip = true`.

Reply:
I see. Thanks!